gamla/functional_generic.py

```diff
@@ -101,6 +168,25 @@ def reraise_and_log(e):
 
 
 def compose_many_to_one(incoming: Iterable[Callable], f: Callable):
+    """Compose an itterable of incoming sync and async functions to a single
+    sink function, in the same order as the incoming functions.
```

Comment:
I think this is not needed (doesn't add anything on top of what's written in 174-176)